src/arrayMethodJoin.js

```diff
@@ -4,8 +4,22 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let joinedString = '';
+
+    for (let i = 0; i < this.length; i++) {
+      if (this[i] === null || this[i] === undefined) {
+        joinedString += '';
+      } else {
+        joinedString += `${this[i]}`;
+      };
+
+      if (i < this.length - 1) {
+        joinedString += `${separator}`;
```

Comment:
same here (`'${}'` - delete it ;3)

Reply:
ty for review, i've already fixed it ^_^